common/services/allocation.js

```diff
@@ -83,8 +83,17 @@ const allocationService = {
   },
   transform({ includeCancelled = false } = {}) {
```

Comment:
So, I think with this change you should be able to strip out all the work this option is doing. It was only really there to try and come up with an accurate number of moves on the dashboard. But if the API is now doing this correctly we can simplify a lot of this logic.

Reply:
Mentioned previously to @solidgoldpig - happy to include an `unfilled` count in the response meta if it means simplifying this code; it's trivial to add in the backend now and won't impact performance. Let me know if you'd like that 😄

Reply:
@smoothcontract That might actually be nice just to avoid us having to calculate in certain situations.

The main purpose of this code though was to try and provide an accurate number for "cancelled" allocations. As with cancelled allocations we needed to not filter out cancelled moves to try and get an accurate original count. Otherwise they all just showed as `0 people`. But I'm guessing that even if the allocation is cancelled, the new meta data will show the number of requested moves before the allocation was cancelled?

Reply:
Will attempt to remove this when the single allocation is updated to return just the uncancelled moves

Reply:
Again, I'd say if it's a separate improvement, maybe we should capture it on the backlog so we can keep track of it?